test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandlerClaimsIdentityTests.cs

```diff
@@ -41,7 +41,7 @@ public void CreateClaimsIdentity_ReturnsCaseSensitveClaimsIdentity_ByDefault()
         [Fact]
         public void CreateClaimsIdentity_ReturnsClaimsIdentity_WithAppContextSwitch()
         {
-            AppContext.SetSwitch(AppContextSwitches.UseClaimsIdentityTypeSwitch, true);
+            AppContextSwitches.UseClaimsIdentityType = true;
```

Comment:
Are we going to have a race condition with public void Create_FromTokenValidationParameters_ReturnsCorrectClaimsIdentity(bool useClaimsIdentity)?

Reply:
it is part of a test collection so it is serialized.